src/components/SecurityUser/AddSecurityUserComponent.jsx

```diff
@@ -35,7 +35,7 @@ class AddSecurityUserComponent extends Component {
   handleSubmit = (event) => {
     event.preventDefault();
 
-    const isValidEmail = this.validateEmail();
+    const isValidEmail = this.state.emailValid;
```

Comment:
This line is unnecessary. You can use the state variable directly in the `if` statement. E.g.
```jsx
if (!this.state.emailValid) {
  return null;
}